charts/spire/charts/spire-server/templates/post-upgrade-hook.yaml

```diff
@@ -51,6 +51,9 @@ metadata:
   annotations:
     "helm.sh/hook": post-upgrade
     "helm.sh/hook-delete-policy": before-hook-creation, hook-succeeded, hook-failed
+    {{- with .Values.podAnnotations }}
```

Comment:
I wonder if these annotations need to be separately configurable from the workload itself? 🤔 

Not completely sure though.